package-lock.json

```diff
@@ -6372,14 +6355,12 @@
         "wrappy": {
           "version": "1.0.2",
           "bundled": true,
-          "dev": true,
-          "optional": true
+          "dev": true
         },
         "yallist": {
           "version": "3.0.3",
           "bundled": true,
-          "dev": true,
-          "optional": true
+          "dev": true
```

Comment:
why are there changes to package-lock in this PR?

Reply:
It seams I used a different version of "npm". I'll fix it.